### PR TITLE
Rocket Buff, I guess

### PR DESCRIPTION
**Pull request by InterroLouis**
## About The Pull Request

Removes smoke from AP rockets and WP rockets, makes WP rockets use fire armor. 
## Changelog
:cl:
tweak: Smoke removed from WP and AP rockets
balance: WP Rockets use Fire Armor
/:cl:

### CODE REVIEW

**Comment by jroinc** [comment]
Do HE rockets have smoke? Cause I'd remove it from them, but leave it on AP.

Cloaked AP SADAR is a BIG meme when it happens, and if the target's lucky enough to not get instagibbed, it should get a chance to get away under cover of smoke.

HE, on the other hand, nah. The ayy should have some health left, so you should be free to chase it's ass.

**Comment by Rohesie** [comment]
Can you reduce the AP rocket damage so it doesn't one-shot an Ancient Empress?

File: code/modules/projectiles/updated_projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -804,20 +804,12 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	max_range = 30
 	damage = 50
 	penetration = 80
-	var/datum/effect_system/smoke_spread/smoke
 
 /datum/ammo/rocket/New()
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/ammo_datums.dm]: This proc is doing nothing. Finish removing it.

Comment (Rohesie) [reply code/modules/projectiles/updated_projectiles/ammo_datums.dm]: This is still a thing.

File: code/modules/projectiles/updated_projectiles/ammo_datums.dm [hunk 2]
```diff
@@ -862,6 +848,7 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 	name = "white phosphorous rocket"
 	hud_state = "rocket_fire"
 	flags_ammo_behavior = AMMO_ROCKET|AMMO_INCENDIARY|AMMO_EXPLOSIVE
+	armor_type = "fire"
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/ammo_datums.dm]:
```suggestion
```
The projectile impact should be brute, not fire.

**Pull request merged**